base_cancel_confirm/model/base_cancel_confirm.py

```diff
@@ -25,6 +27,19 @@ class BaseCancelConfirm(models.AbstractModel):
         help="An optional cancel reason",
     )
 
+    def _cancel_confirm_disabled(self):
+        key = "%s.cancel_confirm_disable" % self._name
+        res = self.env["ir.config_parameter"].get_param(key)
```

Comment:
```suggestion
        res = self.env["ir.config_parameter"].sudo().get_param(key)
```
@kittiu Users other than admin can't work.
![Selection_002](https://user-images.githubusercontent.com/51266019/170621234-ee3a3891-e9f3-4f89-b592-4a0e8cc7aa84.png)

Reply:
OK thanks!